services/prometheus/scrape_configs.go

```diff
@@ -174,68 +175,146 @@ func scrapeConfigForStandardExporter(interval time.Duration, node *models.Node,
 	return cfg, nil
 }
 
-func scrapeConfigForNodeExporter(interval time.Duration, node *models.Node, agent *models.Agent) (*config.ScrapeConfig, error) {
-	return scrapeConfigForStandardExporter(interval, node, nil, agent, []string{})
+func scrapeConfigForNodeExporter(s *models.MetricsResolutions, node *models.Node, agent *models.Agent) ([]*config.ScrapeConfig, error) {
+	hrc := []string{
+		"diskstats",
+		"filefd",
+		"filesystem",
+		"loadavg",
+		"meminfo",
+		"netdev",
+		"netstat",
+		"stat",
+		"time",
+		"vmstat",
+		"meminfo_numa",
+		"textfile(hr)",           // TODO: Implement it!
```

Comment:
`textfile.hr`, see https://jira.percona.com/browse/PMM-3645. Parentheses would require escaping in both query parameters and in a shell.

That ticket also can be done independently, no need to wait for PMM-3645.